File: PPFUV_front/bp2/src/app/entities/predstava/predstava.ts
import { Reditelj } from '../reditelj/reditelj';
import { Glumac } from '../glumac/glumac';
import { Izabran } from '../izabran/izabran';

export class Predstava {
    id: number;
    naziv: string;

    izabran: Izabran;

    reditelji: Array<Reditelj>;
    glumci: Array<Glumac>;
}

File: PPFUV/PPFUV/Model/Predstava.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace PPFUV.Model
{
    public class Predstava
    {
        [Key, DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int id { get; set; }

        public string naziv { get; set; }

        public Izabran izabran { get; set; }

        //public Reditelj reditelj { get; set; }
    }
}
